tests/integration/hello_test.go

```diff
@@ -0,0 +1,86 @@
+package proto
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Libraries from standard lib firtst, than an empty line, than github/whatever imports.

Reply:
done in 83cea2847da28967f0fba35ebd6d03eb3635d805